packages/components-library/src/components/filters/CheckboxFilter.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
-    <div v-if="showSatisfyAllCheckbox" class="query-type-selector">
-      <label class="label-disabled">
+    <div v-if="showSatisfyAllCheckbox" class="text-right mb-2">
```

Comment:
A special class allows users of the component to override the styling, which is sort of nice I think. How about keeping some kind of class label?

Reply:
would like to defer this change untill needed